### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by Jacobichou** [rubynoobs/beaker#7]
Get tests setup for current code base.

**Comment by Jacobichou** [comment]
I haven't forgotten about the tests. I've been working on them, just busy with recent personal things going on! Plus, it's a bit odd having to add tests for code after the fact, especially when the code was written by someone other than yourself, haha.

To write the test suite for the current code base I've needed to come to an understanding of the current code, and what it should be producing/isn't happening as taken from the project outline. A bit of work but all worth it! :D

**Comment by patrickbrown-dev** [comment]
I think everyone here knows that we all have busy lives aside from rubynoobs so not a problem :)

Maybe instead of writing tests for code that has been written you could write some for future code to work on. @n8fischer would be a good person to talk to about that.

In the meantime I really should write tests for the code I lazily wrote w/o tests. :P

**Comment by Jacobichou** [comment]
Well, I've actually written some new code as well. I just hadn't mentioned it because as of yet it isn't meaningful. XD

**Comment by n8fischer** [comment]
Push up a merge request with the new code!

**Comment by patrickbrown-dev** [comment]
True. The worst that can happen is that we deny the pull request. :+1:

**Comment by Jacobichou** [comment]
One thing I've noticed is that the HTML code is a bit scattered. Has someone been putting style changes in the bootstrap CSS file directly?

**Comment by patrickbrown-dev** [comment]
I hope not -- we can pull down a fresh copy of the bootstrap css if so.
